packages/@jsii/kernel/lib/api.ts

```diff
@@ -100,6 +100,19 @@ export interface LoadResponse {
   readonly types: number;
 }
 
+export interface InvokeScriptRequest {
+  readonly pkgname: string;
```

Comment:
I would rather try to make sure we are consistent with how we refer to those concepts... in this case, we refer to *assemblies* (since the value that ought to be passed here is the same that was passed to `LoadRequest.assembly`).

```suggestion
  readonly assembly: string;
```